### PR TITLE
cmds/core/ip: fuzzing test for ip command

### PR DESCRIPTION
**Pull request by RiSKeD**
1. commit: rewrote ip cmd to be testable without testutil.Run()
2. commit: added a fuzzing test with seeds and dictionary to test the command

### CODE REVIEW

File: cmds/core/ip/ip_linux.go [hunk 1]
```diff
@@ -333,62 +333,70 @@ func routedel() error {
...
+		return err
+	}
+	return nil
+}
+
+func main() {
+	flag.Parse()
+	arg = os.Args[1:]
+	if err := run(os.Stdout); err != nil {
+		log.Fatalf("ip: %v", err)
```
Comment (rminnich) [inline cmds/core/ip/ip_linux.go:400]: The standard command does not print the ip: part. I wonder if, for automation, we should just print the error?

**Review by rminnich — APPROVED** [review]
Looks good save for the error message, and on that I'm not certain.

File: cmds/core/ip/ip_linux.go (hunk 1 above)
Comment (RiSKeD) [reply cmds/core/ip/ip_linux.go:400]: Putting the command name first is a convention already established in some of the other commands.
But i don't feel strongly about this.

**Comment by RiSKeD** [comment]
@rminnich Could you take a look at the failing test for the amd64 system? I first thought it to be just flaky, but all reruns failed too and the test does not fail locally...

**Comment by rminnich** [comment]
have you tried the RUNLOCAL script in integration/
it should run exactly as it does in CI. I will try it now.

**Comment by RiSKeD** [comment]
Yes, when trying to pinpoint the problem I ran the RUNLOCAL script on my local system.
And it succeeds on my local amd x86-64 system, hence my confusion.

**Comment by RiSKeD** [comment]
Yes, when trying to pinpoint the problem I ran the RUNLOCAL script on my local system.
And it succeeds on my local amd x86-64 system, hence my confusion.

**Comment by RiSKeD** [comment]
Yes, when trying to pinpoint the problem I ran the RUNLOCAL script on my local system.
And it succeeds on my local amd x86-64 system, hence my confusion.

**Comment by RiSKeD** [comment]
Yes, when trying to pinpoint the problem I ran the RUNLOCAL script on my local system.
And it succeeds on my local amd x86-64 system, hence my confusion.

**Comment by RiSKeD** [comment]
Yes, when trying to pinpoint the problem I ran the RUNLOCAL script on my local system.
And it succeeds on my local amd x86-64 system, hence my confusion.

**Comment by RiSKeD** [comment]
Yes, when trying to pinpoint the problem I ran the RUNLOCAL script on my local system.
And it succeeds on my local x86-64 system, hence my confusion.

**Review by ChriMarMe — APPROVED** [review]

**Pull request merged**